### ISSUE TITLE
empty-heading false positive

### ISSUE THREAD

**Issue by dsturley** [dequelabs/axe-core#39]
`empty-heading` rule throws on a violation on:

```html
<h3 role="presentation">Not empty</h3>
```
